src/style.css

```diff
@@ -1 +1,78 @@
-/* Styles go here */
+body {
+  --color: #00acdc;
+  background: #e5e5e5;
+  font-family: "Roboto", "Open Sans", sans-serif;
+  margin: 0;
+}
+
+.header__container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 1024px;
+  margin: 0 auto;
+  background: #fff;
+  padding: 0 50px;
+}
+
+.header__logo {
+  width: 40px;
+  height: 40px;
+  padding: 10px 0;
+}
+
+.nav__list {
+  display: flex;
+  list-style-type: none;
+  align-items: center;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item {
+  margin-right: 20px;
+  cursor: pointer;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+  text-transform: uppercase;
+  font-style: normal;
```

Comment:
normal is default value for the font-style so this style declaration is redudndant